src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -274,6 +290,9 @@ Environment variables:
 		"only print pachctl's version, but don't make any RPCs to pachd. Useful "+
 		"if pachd is unavailable")
 	rawFlag(versionCmd)
+	versionCmd.Flags().StringVar(&timeoutFlag, "timeout", "default", "If set, "+
```

Comment:
Updated the flag description. Basically, if the flag is unset, it uses the default timeout in `client.go` (currently 30s), but if set to `0s`, then it overrides the 30s with 0s, which means "no timeout".

I basically used `"default"` as the value so that if we wanted to change the default timeout in client.go, we wouldn't have to remember to change it here too (in general I liked the idea that the flag would only do anything if it's explicitly set)